wizard/src/utils/detect_place.ts

```diff
@@ -260,6 +298,53 @@ export class PlaceDetector {
     return null;
   }
 
+  /**
+   * State is detected with high confidence if > MIN_HIGH_CONF_DETECT of the
+   * non-null column values match one of the state format (property) arrays.
+   * If state is not detected, null is returned.
+   * If state is detected, the TypeProperty is returned.
+   *
+   * @param column: an array of strings representing the column values.
+   *
+   * @return the TypeProperty object or null if nothing can be determined with
+   *  high confidence.
+   */
+  detectStateHighConf(column: Array<string>): TypeProperty {
+    let numValid = 0;
+
+    const counters = new Map<string, number>();
+    counters["name"] = 0;
+    counters["isoCode"] = 0;
+    counters["fips52AlphaCode"] = 0;
+    for (const cVal of column) {
+      if (cVal == null) {
+        continue;
+      }
+      const v = toAlphaNumeric(cVal);
+      numValid++;
+
+      if (this.stateNames.has(v)) {
+        counters["name"]++;
+      } else if (this.stateISO.has(cVal.toLowerCase())) {
+        // Note: this matches lower case only.
```

Comment:
does casing matter for stateNames and fips code?

Reply:
Sorry, they all use the helper function which deletes non-Alpha Numeric chars but also converts to lowercase. I have renamed the function to toAlphaNumericAndLower() from toAlphaNumeric(). Hope this makes the contrast clearer.